mne/viz/_3d.py

```diff
@@ -1441,12 +1440,12 @@ def _smooth_plot(this_time, params):
     curv_ave = np.mean(params['curv'][faces], axis=1).flatten()
     # matplotlib/matplotlib#11877
     facecolors = polyc._facecolors3d
-
-    to_blend = color_ave > params['ctrl_pts'][0] / vmax
-    facecolors[to_blend, :3] = ((1 - transp) *
-                                greymap(curv_ave[to_blend])[:, :3] +
-                                transp * cmap(color_ave[to_blend])[:, :3])
-    facecolors[~to_blend, :3] = greymap(curv_ave[~to_blend])[:, :3]
+    colors = cmap(color_ave)
+    # alpha blend
+    colors[:, :3] *= colors[:, [3]]
+    colors[:, :3] += greymap(curv_ave)[:, :3] * (1.  - colors[:, [3]])
+    colors[:, 3] = 1.
+    facecolors[:] = colors
```

Comment:
@larsoner can you explain what you fixed?

Reply:
The alpha blending was wrong before

Reply:
(This is now equivalent to standard OpenGL blending like ONE_MINUS_SRC_ALPHA)